### DISCUSSION TITLE
Registering taxa at IPNI

### DISCUSSION THREAD

**Discussion by choess** [pteridogroup/ppg#83] [General]
This is a little off-topic from our usual business, but I'm not sure this is as widely known as it should be for those publishing taxonomic novelties.

IPNI now has a form for registering new names, combinations, and replacement names: https://ipni.org/#index__registration_start If you have published or are about to publish a novelty (I believe you can embargo names in the latter for later release but I have not tried this feature), you can enter it here and it will typically be reviewed and entered into IPNI within a day or so, rather than waiting for IPNI staff to pick up on the publication itself. A few journals (notably Phytotaxa) will automatically enter novelties into IPNI through this process without author intervention.

Names typically flow from IPNI into other downstream taxonomies (WFO, POWO, World Ferns, etc.) so this is useful if you want to accelerate the availability of your novelty for other uses.

**Comment by joelnitta** [comment]
Thanks for the tip @choess ! This is even more relevant now, as we will be soon merging with WFO, and as you mentioned WFO draws its names from IPNI. It will definitely make our workflow smoother if any new names are submitted to IPNI as soon as possible.
